apps/fp-frontend-app/src/fagsakprofile/FagsakProfileIndex.tsx

```diff
@@ -38,7 +38,7 @@ const finnFagsakMarkeringTekst = (fagsak: Fagsak): string[] => {
   if (!fagsak.fagsakMarkeringer) {
     return [];
   }
-  return fagsak.fagsakMarkeringer.map(m => m.kortNavn).filter(navn => !!navn) as string[];
+  return fagsak.fagsakMarkeringer.map(m => m.kortNavn) as string[];
```

Comment:
Kan ikkje kortNavn vera undefined? Da vil ein her kunna laga eit array av undefines. Er det meininga?

Reply:
@tor-nav det ene tilfellet er fjernet backend. Men vi kan godt beholde for sikkhets skyld

Reply:
Revertet denne